datatypes/src/primitives/time_step.rs

```diff
@@ -905,6 +939,21 @@ mod tests {
         )
     }
 
+    #[test]
+    fn time_snap_0() {
+        let time_snapper = TimeStep {
+            granularity: TimeGranularity::Months,
+            step: 1,
+        };
+
+        assert_eq!(
+            time_snapper
+                .snap_relative(1_388_534_400_000.into(), 0.into())
```

Comment:
could you add a comment what this number represents?

Reply:
3300ec0b476320d9e6705ea58230fe3647ce4283